src/core/main.cpp

```diff
@@ -257,7 +257,58 @@ int main( int argc, char * * argv )
 {
 	using namespace lmms;
 
-#ifdef LMMS_DEBUG_FPE
+	bool coreOnly = false;
+	bool fullscreen = true;
+	bool exitAfterImport = false;
+	bool allowRoot = false;
+	bool renderLoop = false;
+	bool renderTracks = false;
+	QString fileToLoad, fileToImport, renderOut, profilerOutputFile, configFile;
+
+	// first of two command-line parsing stages
+	for (int i = 1; i < argc; ++i)
+	{
+		QString arg = argv[i];
+
+		if (arg == "--help" || arg == "-h")
+		{
+			printHelp();
+			return EXIT_SUCCESS;
+		}
+		else if (arg == "--version" || arg == "-v")
+		{
+			printVersion(argv[0]);
+			return EXIT_SUCCESS;
+		}
+		else if (arg == "render" || arg == "--render" || arg == "-r" )
```

Comment:
Actual if else ladder should start here.

Reply:
See above for switch/case.